### PR TITLE
Removing all dashboard references

### PR DESCRIPTION
**Pull request by richmccartney**
#### What's this PR do?

Removes all remaining references to dashboard in the develop branch
#### Where should the reviewer start?

Pull the branch and check all documentation doesn't have references to dashboard

### CODE REVIEW

**Review by anthonydillon — CHANGES_REQUESTED** [review]
The titles of both the `CONTRIBUTING.md` and `README.md` still have the new name.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ So, you'd like to contribute to the Dashboard Vanilla Theme? Great!
```
Comment (richmccartney) [inline CONTRIBUTING.md:5]: Can you confirm this is fine now?

**Comment by richmccartney** [comment]
@anthonydillon can you confirm this is fine now?

**Comment by anthonydillon** [comment]
No the issue above still exists.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -5,9 +5,9 @@ So, you'd like to contribute to the Dashboard Vanilla Theme? Great!
 
 ## Bugs and issues
 
-We use the [Issues tracker](https://github.com/ubuntudesign/dashboard-vanilla-theme/issues) to track all our bugs and feature requests, collectively known as 'issues'.
+We use the [Issues tracker](https://github.com/ubuntudesign/maas-gui-vanilla-theme/issues) to track all our bugs and feature requests, collectively known as 'issues'.
 
-When [submitting a new issue](https://github.com/ubuntudesign/dashboard-vanilla-theme/issues/new), please check that it hasn't already been raised by someone else. We've provided a template for new issues which will help you structure your issue to ensure it can be picked up and actioned with minimum fuss.
+When [submitting a new issue](https://github.com/ubuntudesign/maas-gui-vanilla-theme/issues/new), please check that it hasn't already been raised by someone else. We've provided a template for new issues which will help you structure your issue to ensure it can be picked up and actioned with minimum fuss.
```
Comment (richmccartney) [inline CONTRIBUTING.md:10]: @anthonydillon can you not see these changes at all in the files?

File: README.md [hunk 3]
```diff
@@ -20,6 +20,6 @@ You can override any of the settings in [_settings.defaults.scss](scss/_settings
 
 ## Demo
 
-To see the latest changes and updates live please view the [demo page](http://ubuntudesign.github.io/dashboard-vanilla-theme/)
+To see the latest changes and updates live please view the [demo page](http://ubuntudesign.github.io/maas-gui-vanilla-theme/)
```
Comment (richmccartney) [inline README.md:23]: @anthonydillon same with this change?

**Comment by anthonydillon** [comment]
Yea I can see them but I'm referring to the title.

# Contributing to Dashboard Vanilla Theme

**Comment by anthonydillon** [comment]
Now it is just the title of the Readme.

# Dashboard Vanilla Theme

**Review by anthonydillon — APPROVED** [review]

**Comment by anthonydillon** [comment]
Looks good.

**Pull request merged**